python/cudf/cudf/core/column_accessor.py

```diff
@@ -653,10 +654,13 @@ def rename_column(x):
                 return x
 
             if level is None:
-                raise NotImplementedError(
-                    "Renaming columns with a MultiIndex and level=None is"
-                    "not supported"
+                warnings.warn(
+                    "Renaming columns with MultiIndex assuming level=0. "
```

Comment:
Does pandas raise this same warning? I would be good to validate comparing a public cudf API call vs the same public pandas API call

Reply:
No Pandas doesn't.